macros/contexts/contextBoolean.pl

```diff
@@ -0,0 +1,160 @@
+## contextBoolean.pl
+
+sub _contextBoolean_init { context::Boolean::Init() }
+
+package context::Boolean;
+
+sub Init {
+	my $context = $main::context{Boolean} = Parser::Context->getCopy("Numeric");
+	$context->{name} = "Boolean";
+	$context->{value}{Formula} = "context::Boolean::Formula";
+
+	## Disable unwanted operations and functions
+	$context->functions->disable('All');
+	#    $context->parens->undefine('<');
+	$context->operators->undefine('-', '/', ' /', '/ ', '//', '^', '**', 'u+', '!');
+	$context->operators->redefine('><');
+
+	## Re-enable necessary operations
+	$context->operators->set(
+		'+'   => { class => "Parser::BOP::or",  string => ' or ',  TeX => '\vee ',   perl => '||' },
+		'or'  => { class => "Parser::BOP::or",  string => ' or ',  TeX => '\vee ',   perl => '||' },
+		'and' => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		'*'   => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		' *'  => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		'* '  => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		# implicit multiplication must have string '*' for Parser::Op (L#314)
```

Comment:
The comment here doesn't match what happens in the next line.

Reply:
It does. The next line assigns the whitespace operator (implicit multiplication) with text string ' *', so that when pushed onto the operator stack, it ends up as an 'and'. Refer to line 314 of Parser::Op to see how this works, the string output gets used in this case.

Though now, with the inclusion of 'and' as a literal operator token, I think it's probably safe to remove this comment and align the whitespace operator with the other 'and' operations.

This was the source of the issue I described in last week's dev meeting -- implicit multiplication was causing Formula creation to fail (ultimately due to an error parsing the string output from the original whitespace token when building the formula tree). This was before I had included 'and' and 'or' as literal operator tokens.

Reply:
I was confused why the comment uses '*' (no space) but the code in the next line uses ' *' (with space). Also not sure what `Parser::Op` is. There is `lib/Parser/BOP.pm`, is that what this refers to? But then for me line 314 seems off.

Reply:
Parser.pm line 314, inside `sub Op` -- as you note, there is no `Parser/Op.pm`